client/app/telecom/telephony/billingAccount/administration/optionsGroup/telecom-telephony-billing-account-administration-options-group.html

```diff
@@ -16,134 +9,57 @@ <h1 data-translate="telephony_options_group_title"></h1>
-                                           data-uib-btn-radio="false"
-                                           data-translate="common_no_capitalized">
-                                    </label>
-                                </div>
-                            </div>
-                        </div>
-                    </div> <!-- END OF OVERRIDE DISPLAYED NUMBER -->
-                    <!-- /.widget-presentation -->
+        <form name="OptionsGroupCtrl.optionsGroupSettings"
+              data-ng-submit="OptionsGroupCtrl.changeSettings()">
 
-                    <!-- DISPLAY ON INCOMING CALL -->
-                    <div class="widget-presentation">
-                        <header class="widget-presentation-header">
-                            <h2 class="widget-presentation-title"
-                                data-translate="telephony_billing_account_administration_options_group_displayOnIncomingCall">
-                            </h2>
-                        </header>
-                        <div class="widget-presentation-content">
-                            <div class="oui-box mb-3"
-                                 data-translate="telephony_billing_account_administration_options_group_displayOnIncomingCall_info">
-                            </div>
-                            <div class="form-group">
-                                <label for="displayOnIncomingCall"
-                                       class="control-label"
-                                       data-translate="telephony_billing_account_administration_options_group_displayOnIncomingCall_label">
-                                </label>
-                                <div class="btn-group d-block clearfix">
-                                    <label class="btn btn-default"
-                                           data-ng-model="OptionsGroupCtrl.optionsGroupForm.user.lineDescriptionPolicies.displayOnIncomingCall"
-                                           data-uib-btn-radio="true"
-                                           data-translate="common_yes_capitalized">
-                                    </label>
-                                    <label class="btn btn-default"
-                                           data-ng-model="OptionsGroupCtrl.optionsGroupForm.user.lineDescriptionPolicies.displayOnIncomingCall"
-                                           data-uib-btn-radio="false"
-                                           data-translate="common_no_capitalized">
-                                    </label>
-                                </div>
-                            </div>
-                        </div>
-                    </div> <!-- END OF DISPLAY ON INCOMING CALL -->
-                    <!-- /.widget-presentation -->
+            <oui-field data-label="{{:: 'telephony_billing_account_administration_options_group_hiddenExternalNumber_label' | translate }}">
+                <oui-radio name="hiddenExternalNumber"
```

Comment:
You should wrap oui-radio in a oui-radio-group http://master.ui-kit.ovh/#!/oui-angular/radio-group

Reply:
This almost a copy/paste from http://master.ui-kit.ovh/#!/oui-angular/field